packages/react/src/patterns/sections/CardSection/CardSection.js

```diff
@@ -67,6 +67,7 @@ const CardSection = ({ heading, theme, cards, cta }) => {
   return (
     <ContentSection
       heading={heading}
+      autoid={otherProps.autoid}
```

Comment:
Approved, but I'm wondering if `autoid` should be a first class prop for the `CardSection`, but default as `null` under `defaultProps`?

Reply:
Nevermind, I think we'll get the same effect with this implementation, right?

Reply:
yep it'll default to `null` anyways